tests/test-module-test.js

```diff
@@ -159,3 +159,20 @@ test("needs is not needed (pun intended) when integration is true", function() {
   var otherComponent = this.container.lookup('component:not-the-subject');
   ok(otherComponent, 'another component can be resolved when integration is true');
 });
+
+test("throws an error when declaring integration: true and needs in the same module", function() {
+  expect(1);
```

Comment:
(sorry) - It looks like 2 assertions are made in the `beforeSetup` just above here causing this to error (because we actually make 3 assertions).